### PR TITLE
feat(ingest): rest_emitter - Adding option to disable ssl

### PR DESCRIPTION
**Pull request by szalai1**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by shirshanka — APPROVED** [review]
LGTM!

File: metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def __init__(self, ctx: PipelineContext, config: DatahubRestSinkConfig):
             retry_max_times=self.config.retry_max_times,
             extra_headers=self.config.extra_headers,
             ca_certificate_path=self.config.ca_certificate_path,
+            disable_ssl_verification=config.disable_ssl_verification,
```
Comment (shirshanka) [inline metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py:55]: nit: maybe use `self.config.` to align with previous lines?

**Comment by shirshanka** [comment]
Fixes #5542

File: metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py (hunk 1 above)
Comment (szalai1) [inline metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py:55]:
```suggestion
            disable_ssl_verification=self.config.disable_ssl_verification,
```

Comment (szalai1) [reply metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py]: fixed

**Comment by szalai1** [comment]
@treff7es added the field to the `dhub` schema generation

**Review by treff7es — APPROVED** [review]
Looks good, congrats!

**Pull request merged**